Mathlib/Analysis/SpecialFunctions/Trigonometric/Complex.lean

```diff
@@ -98,6 +98,22 @@ theorem sin_eq_sin_iff {x y : ℂ} :
   refine' exists_congr fun k => or_congr _ _ <;> refine' Eq.congr rfl _ <;> field_simp <;> ring
 #align complex.sin_eq_sin_iff Complex.sin_eq_sin_iff
 
+theorem cos_eq_one_iff {x : ℂ} : cos x = 1 ↔ ∃ k : ℤ, k * (2 * π) = x := by
```

Comment:
Why don't you write it as
```suggestion
theorem cos_eq_one_iff {x : ℂ} : cos x = 1 ↔ ∃ k : ℤ, x = k * (2 * π) := by
```
with the `x` on the left, like in the other statements of the file? Same thing below.

Reply:
It aligns with [Real.cos_eq_one_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.html#Real.cos_eq_one_iff). Probably, we should have a convention about this. I'll ask on Zulip.

Reply:
I agree with the current spelling that Yury uses, but an even better one would use `AddCommGroup.ModEq`